tests/integration/it_locations_test.py

```diff
@@ -37,16 +37,34 @@ def test_create(client):
 
 
 def test_edit(client):
-    assert client.get("/locations/edit/1").status_code == HTTPStatus.OK
+    location_original = factories.LocationFactory()
+    location_edited = {
```

Comment:
@vladarefiev wouldn't it be better to using `factories.LocationFactory().build()` to get a new instance of Location (without saving it to the db) and passing that to the edit function?

Reply:
@emilianodellacasa I have just tried to do it here, but I don't like how it looks like:
```
location_original = factories.LocationFactory()
    location_edited = factories.LocationFactory.build()
    # convert reservation instance to dict
    location_edited_dict = {
        column.name: str(getattr(location_edited, column.name))
        for column in location_edited.__table__.columns
        if column.name not in ['synchronized_on', 'poster_id', 'id']
    }
```
we should do smth like this. WDYT?

Reply:
@vladarefiev just an idea: adding a `edit_fields_dict` method to `LocationFactory` bulding itself and returning the dict, the way you wrote with excluded fields. This way, that logic stays inside the factory and you would just write location_edited_dict = factories.LocationFactory.edit_fields_dict().

Could it work for you? Naming may change if you find something better